app/Helpers/CountriesHelper.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Helpers;
+
+use PragmaRX\CountriesLaravel\Package\Facade as Countries;
+
+class CountriesHelper
+{
+    /**
+     * Get list of countries.
+     *
+     * @return string
+     */
+    public static function getAll()
+    {
+        $countries = Countries::all()->map(function ($item, $key) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the unused function parameter "$key". ([php:S1172](https://sonarcloud.io/organizations/monicahq/rules#rule_key=php%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=monica&pullRequest=1117&issues=AWLQmTCTjr7wspRoor_3&open=AWLQmTCTjr7wspRoor_3)

[ISSUE_KEY]: AWLQmTCTjr7wspRoor_3